corehq/apps/hqwebapp/static/hqwebapp/less/_hq/navs.less

```diff
@@ -61,4 +61,10 @@
 
 .nav > li > a {
   color: @cc-brand-low;
+  display: -webkit-box;
+  -webkit-line-clamp: 2;
+  -webkit-box-orient: vertical;
+  overflow: hidden;
+  word-break: break-all;
+  text-overflow: ellipsis;
```

Comment:
hmm, actually, can you make this change specific to the navigation class for this use case? What you are changing here affects ALL nav links. not a good idea as we do want overflow for some of them